bundle/edu.gemini.p1monitor/src/main/scala/edu/gemini/p1monitor/config/P1MonitorConfig.scala

```diff
@@ -104,14 +104,14 @@ class P1MonitorConfig(ctx: BundleContext) {
   }
 
   private def toTemplate(s: String): P1PDF.Template = s match {
-    case "ar" | "br" | "ca" | "lp"               => P1PDF.GeminiDARP
-    case "kr" | "uh"                             => P1PDF.GeminiStandard
-    case "cfh" | "subaru" | "sip" | "keck"       => P1PDF.GeminiStandard
-    case "ds" | "dt" | "sv" | "pw" | "gt" | "lp" => P1PDF.GeminiStandard
-    case "cl"                                    => P1PDF.CL
-    case "us"                                    => P1PDF.NOIRLabDARP
-    case "ft"                                    => P1PDF.GeminiDefaultNoInvestigatorsList
-    case _                                       => P1PDF.GeminiDefaultListAtTheEnd
```

Comment:
Was the `da` entry for SIP and Subaru default?  I assume that now there is no `<template>` element it uses the `name` attribute or something?